README.md

```diff
@@ -7,42 +7,32 @@ This library adds the required files to a DDEV-Local configured Drupal 9 site so
 ## Instructions for use
 
 1.  Start with a Drupal 9 site (based on the drupal/recommended-project Composer template) with a DDEV-Local configuration file, committed to a GitHub.com repository. 
-2.  Add the following to the Drupal site's composer.json file "repositories" section:
+1.  Add the following to the Drupal site's composer.json file "drupal-scaffold" section:
+    ```
+    "allowed-packages": [
+        "ultimike/ddevdrupalgitpod"
+    ]
+    ```
 
-```
-{
-    "type": "vcs",
-    "url": "https://github.com/ultimike/ddevdrupalgitpod"
-}
-```
+1.  `composer config repositories.ddevdrupalgitpod '{"type": "vcs", "url": "https://github.com/ultimike/ddevdrupalgitpod.git"}'`
```

Comment:
It looks like this is the main change in this pull request, correct? If so, let's just leave it as-is for now as once we have something that is working a bit more cleanly, we can publish this on packagist then be able to remove this step completely. Cool?